haystack/reader/farm.py

```diff
@@ -394,6 +394,11 @@ def eval(
         :param doc_index: Index/Table name where documents that are used for evaluation are stored
         """
 
+        if self.top_k_per_candidate != 4:
+            logger.warning(f"Performing Evaluation using top_k_per_candidate = {self.top_k_per_candidate} \n"
+                           f"and consequently, QuestionAnsweringPredictionHead.n_best = {self.top_k_per_candidate + 1}. \n"
+                           f"This deviates from FARM's default where QuestionAnsweringPredictionHead.n_best = 5")
```

Comment:
- What is the impact of this? As a user, it is unclear to me why this might be a problem. Is it only the `top_k_accuracy` metric that will be different to eval in FARM? If yes, I would rather make it an info than a warning.